### ISSUE TITLE
latest update breaks .launch()

### ISSUE THREAD

**Issue by ehutt** [gradio-app/gradio#43]
I just upgraded from gradio 1.0.7 to 1.1.0 and now I can't launch my demo which was working before. 

When I call:

`gr.Interface(fn=ask, inputs=["textbox", "text"], outputs="text").launch()`

I get the following error: 

```Traceback (most recent call last):
  
File "/Users/elhutton/opt/anaconda3/envs/agent-assist/lib/python3.7/site-packages/IPython/core/interactiveshell.py", line 3343, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  
File "<ipython-input-27-5dcf77e302d7>", line 1, in <module>
    gr.Interface(fn=ask, inputs=["textbox", "text"], outputs="text").launch()
  
File "/Users/elhutton/opt/anaconda3/envs/agent-assist/lib/python3.7/site-packages/gradio/interface.py", line 265, in launch
    server_port=self.server_port)

ValueError: too many values to unpack (expected 2)
```

I am running on MacOS Catalina 10.15.6, working in a conda virtual env with python 3.7.7. I upgraded via `pip install gradio --upgrade` 

Please help! Thanks

**Comment by abidlabs** [comment]
Hi Elizabeth, thanks for pointing this out. We are looking into it.

Abubakar Abid
Department of Electrical Engineering
Stanford University || Paul & Daisy Soros Fellow
@abidlabs <https://twitter.com/abidlabs> || *abidlabs.github.io
<http://abidlabs.github.io>*

**Comment by abidlabs** [comment]
Fixed in latest version, 1.1.4 (it took 4 tries lol). Thanks again for bringing it to our attention!
